workflow-manager/main.go

```diff
@@ -340,13 +350,30 @@ func (b *bucket) listFilesS3(ctx context.Context) ([]string, error) {
 		WithRegion(region).
 		WithCredentials(credentials)
 	svc := s3.New(sess, config)
-	resp, err := svc.ListObjectsV2(&s3.ListObjectsV2Input{Bucket: aws.String(bucket)})
-	if err != nil {
-		return nil, fmt.Errorf("Unable to list items in bucket %q, %w", b.bucketName, err)
-	}
 	var output []string
-	for _, item := range resp.Contents {
-		output = append(output, *item.Key)
+	var nextContinuationToken string = ""
+	for {
+		// We choose a lower number than the default max of 1000 to ensure we exercise the
+		// cursoring case regularly.
+		var hundred int64 = 100
```

Comment:
nit: You could use [`aws.Int64`](https://godoc.org/github.com/aws/aws-sdk-go-v2/aws#Int64) and avoid this